spec/uap_clj/common_spec.clj

```diff
@@ -1,3 +1,27 @@
-(ns uap-clj.common-spec)
+(ns uap-clj.common-spec
+  (:require [clojure.java.io :as io :refer [resource]]
+            [clj-yaml.core :refer [parse-string default-loader-options]])
+  (:import [org.yaml.snakeyaml LoaderOptions]))
 
-(def unknown-ua "Unknown new useragent in the wild/v0.1.0")
+(def ^:const unknown-ua "Unknown new useragent in the wild/v0.1.0")
+
+(defn default-codepoint-limit
+  []
+  (let [options (LoaderOptions.)]
+    (.getCodePointLimit options)))
+
+(defn snakeyaml-loader-options
+  ^LoaderOptions [n]
+  (let [options (LoaderOptions.)]
+    (.setCodePointLimit options n)
+    options))
+
+(defn load-fixture
+  [f]
+  (let [raw-file (slurp (io/resource f))
+        file-size (count raw-file)]
+    (if (>= file-size (default-codepoint-limit))
+      (:test_cases
+        (with-redefs-fn {#'clj-yaml.core/default-loader-options #(snakeyaml-loader-options file-size)}
```

Comment:
Normally I'd use `with-redefs` since it's syntactically nicer and the usecase for `with-redefs-fn` is mainly if you have a dynamically created binding-map?